blueos_repository/consolidate.py

```diff
@@ -146,9 +170,10 @@ async def run(self) -> None:
                         continue
                     raw_labels = await self.registry.fetch_labels(f"{repository.docker}:{tag}")
                     permissions = raw_labels.get("permissions", None)
-                    website = raw_labels.get("website", None)
+                    links = raw_labels.get("links", {})
+                    website = links.pop("website", raw_labels.get("website", None))
                     authors = raw_labels.get("authors", None)
-                    docs = raw_labels.get("docs", None)
+                    docs = links.pop("docs", links.pop("documentation", raw_labels.get("docs", None)))
```

Comment:
why the main "docs" labels is not the priority ?

Reply:
Documentation is just a URL, so it should go as an entry within the new "links" label. If `links` does not contain "docs" or "documentation" then there's a fallback check for the old structure that had "docs" as an independent label.

Reply:
We should add something on CI to avoid people using the deprecated `docs` from the old structure. That can be done in future PRs, but we should at least create an issue to keep a note. 
Besides that, can you add a comment on the code explaining it as well ?
There is no comments on code and in the commit about this change.

Reply:
> We should add something on CI to avoid people using the deprecated `docs` from the old structure. That can be done in future PRs, but we should at least create an issue to keep a note.

I've added a note to #30 



> Besides that, can you add a comment on the code explaining it as well ?

Done



> There is no comments on code and in the commit about this change.

I was originally considering the point about 'links' in the commit description to be sufficient, since I don't think this is a particularly consequential or important change - the field isn't currently being used in BlueOS, and most existing extensions have no value for it.